### PR TITLE
Simplify minikube install

### PR DESCRIPTION
**Pull request by rajansandeep**
Simplify Kubernetes setup:

- We don't need to maintain a separate CoreDNS manifest and reinstall it since CoreDNS is now the default in Kubernetes.

- Use `kubectl patch` to update the docker image and deployment.

### CODE REVIEW

File: build/kubernetes/coredns_deployment_patch.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    spec:
+      containers:
+      - name: coredns
+        image: localhost:5000/coredns
+      volumes:
+      - name: config-volume
+        configMap:
+          name: coredns
+          items:
+          - key: Corefile
```
Comment (chrisohaver) [inline build/kubernetes/coredns_deployment_patch.yaml]: i think we can remove this key

Comment (rajansandeep) [reply build/kubernetes/coredns_deployment_patch.yaml]: If I keep the `Zonefile` Key and path only, the deployment is patched by updating `Corefile` with `Zonefile`. I need to keep both to be able to update the deployment with both values.

**Comment by rajansandeep** [comment]
@chrisohaver All working!

**Pull request merged**